Add FreeBSD to the list of supported operating systems.

At the moment only the "amd64" architecture is defined here but it should be
possible to add more architectures here.

--- local_repos/rules_nodejs/toolchains/node/BUILD.bazel.orig	2021-02-20 14:13:32 UTC
+++ local_repos/rules_nodejs/toolchains/node/BUILD.bazel
@@ -26,6 +26,14 @@ platform(
 )
 
 platform(
+    name = "freebsd_amd64",
+    constraint_values = [
+        "@bazel_tools//platforms:freebsd",
+        "@bazel_tools//platforms:x86_64",
+    ],
+)
+
+platform(
     name = "linux_amd64",
     constraint_values = [
         "@bazel_tools//platforms:linux",
@@ -78,6 +86,7 @@ alias(
     name = "toolchain",
     actual = select({
         "@bazel_tools//src/conditions:darwin": "@nodejs_darwin_amd64_config//:toolchain",
+        "@bazel_tools//src/conditions:freebsd_x86_64": "@nodejs_freebsd_amd64_config//:toolchain",
         "@bazel_tools//src/conditions:linux_aarch64": "@nodejs_linux_arm64_config//:toolchain",
         "@bazel_tools//src/conditions:linux_s390x": "@nodejs_linux_s390x_config//:toolchain",
         "@bazel_tools//src/conditions:linux_x86_64": "@nodejs_linux_amd64_config//:toolchain",
@@ -92,6 +101,7 @@ alias(
     name = "node_bin",
     actual = select({
         "@bazel_tools//src/conditions:darwin": "@nodejs_darwin_amd64//:node_bin",
+        "@bazel_tools//src/conditions:freebsd_x86_64": "@nodejs_freebsd_amd64//:node_bin",
         "@bazel_tools//src/conditions:linux_aarch64": "@nodejs_linux_arm64//:node_bin",
         "@bazel_tools//src/conditions:linux_s390x": "@nodejs_linux_s390x//:node_bin",
         "@bazel_tools//src/conditions:linux_x86_64": "@nodejs_linux_amd64//:node_bin",
@@ -99,6 +109,16 @@ alias(
         "//conditions:default": "@nodejs_linux_amd64//:node_bin",
     }),
     visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "node_freebsd_amd64_toolchain",
+    target_compatible_with = [
+        "@bazel_tools//platforms:freebsd",
+        "@bazel_tools//platforms:x86_64",
+    ],
+    toolchain = "@nodejs_freebsd_amd64_config//:toolchain",
+    toolchain_type = ":toolchain_type",
 )
 
 toolchain(
